### PR TITLE
Change Default from GraphDB to InMemory Knowledge Graph

### PR DESCRIPTION
**Pull request by brandenchan**
Addresses #346

### CODE REVIEW

**Comment by brandenchan** [comment]
@julian-risch Could you confirm whether the section about name spaces is also relevant to InMemoryKnowledgeGraph?

File: docs/latest/guides/knowledge_graph.mdx [hunk 1]
```diff
@@ -6,45 +6,23 @@ Haystack allows loading and querying knowledge graphs. In particular, Haystack c
...
+InMemoryKnowledgeGraph is a triple store similar to Haystack's document stores.
+It does not require any extra installation or setup beyond installing Haystack.
 
-`docker run -d -p 7200:7200 --name graphdb-instance-tutorial docker-registry.ontotext.com/graphdb-free:9.4.1-adoptopenjdk11`
+`kg = InMemoryKnowledgeGraph(index="tutorial_10_index")`
 
-By default, GraphDBKnowledgeGraph connects to a GraphDB instance running on localhost at port 7200.
-Similar to Haystack's ElasticsearchDocumentStore, the only additional setting needed is an index name.
-(Note that GraphDB internally calls these indices repositories.)
+Indices can be deleted and created with `InMemoryKnowledgeGraph.delete_index()` and `InMemoryKnowledgeGraph.create_index(config_path)`.
```
Comment (julian-risch) [inline docs/latest/guides/knowledge_graph.mdx]: `create_index()` doesn't have a `config_path` but only an optional `index` parameter (just as the `delete_index()` method). I would leave out the parameter of `create_index()` here.

File: docs/latest/guides/knowledge_graph.mdx [hunk 2]
```diff
@@ -78,7 +56,8 @@ Internally, Text2SparqlRetriever uses a pre-trained BART model to translate text
 ## Trying Question Answering on Knowledge Graphs with Custom Data
 
 If you want to use your custom data you would first need to have your custom knowledge graph in the format of a .ttl file.
-You can load your custom graph and execute SPARQL queries with `Text2SparqlRetriever._query_kg(sparql_query)`. To allow the use of abbreviations of namespaces, GraphDBKnowledgeGraph needs to know about them:
+You can load your custom graph and execute SPARQL queries with `Text2SparqlRetriever._query_kg(sparql_query)`.
+To allow the use of abbreviations of namespaces, GraphDBKnowledgeGraph needs to know about them:
```
Comment (julian-risch) [inline docs/latest/guides/knowledge_graph.mdx]: Maybe we could emphasize here that setting prefixes explicitly is only necessary with `GraphDBKnowledgeGraph` but not with `InMemoryKnowledgeGraph`. The latter imports them automatically from the ttl file. I'd say that the tutorial notebook also could be clearer on this topic.

**Review by julian-risch — CHANGES_REQUESTED** [review]
Thanks for picking this up so quickly! I left to comments with requests for changes. Everything else looks very good to me already! 👍

**Pull request merged**